src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -84,10 +84,6 @@
 }
 
 @keyframes jw-float-to-top {
```

Comment:
get rid of this because it doesn't do anything now.

Reply:
Can we do a transition on top or translateY to go from current pos (or 0) to the magic number.